trunk/NDHMS/HYDRO_drv/module_HYDRO_drv.F

```diff
@@ -1816,82 +1816,42 @@ end subroutine lsm_input
 end module module_HYDRO_drv
 
 ! stop the job due to the fatal error.
```

Comment:
Are we missing some use statements now that this has been moved out from module_HYDRO_drv.F?

Reply:
Any place that calls hydro_stop() should have the required use statements added by this pull request.

Reply:
`module_HYDRO_drv.F` doesn't have a use statement for it but it does compile... 

There must be a nested `use` that brings it into the namespace, but it's not immediately obvious where it's coming from. In any case, yes, any place that calls `hydro_stop()` should have an _explicit_ `use` statement